dcsazure_Databricks_to_Databricks_mask_pl/dcsazure_Databricks_to_Databricks_mask_pl.json

```diff
@@ -1498,16 +1185,15 @@
             "     store: 'restservice',",
             "     timeout: 30,",
             "     requestInterval: 0,",
-            "     headers = ['Run-Id' -> $runId, 'Field-Algorithm-Assignment' -> $DF_FIELD_ALGORITHM_ASSIGNMENT],",
+            "     headers = ['Run-Id' -> $runId, 'Field-Algorithm-Assignment' -> $DF_FIELD_ALGORITHM_ASSIGNMENT, 'Fail-On-Non-Conformant-Data' -> iif($DF_FAIL_ON_NONCONFORMANT_DATA,'true','false'), 'Field-Date-Format' -> $DF_FIELD_DATE_FORMAT],",
             "     httpMethod: 'POST',",
-            "     entity: '/v1/masking/batchMaskByColumn',",
+            "     entity: '/maaas-dev04/v1/masking/batchMaskByColumn',",
```

Comment:
I think this should stay pointing to prod.

Reply:
Good catch